e2e/cypress/support/api_commands.js

```diff
@@ -234,6 +234,22 @@ Cypress.Commands.add('apiUnpinPosts', (postId) => {
     });
 });
 
+/**
+ * Get a post
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} post_id - The post id of the post
+ */
+Cypress.Commands.add('apiGetPost', (postId) => {
```

Comment:
Not sure this is needed if we verify the current user from UI.

Reply:
Ok. I've adopted your code snippet and verify the user on the UI instead of using the API.